examples/v4/task-metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "Containers": [
```

Comment:
This entire file isn't needed. The feature will inject the 'container' metadata into the Task metadata response for each container (i.e. it will be in the list of containers here). This file is only for top level fields describing the task that are not in the containers list. This is something to mention in the docs as well.

Reply:
oh i got you